src/xo-app/host/tab-network.js

```diff
@@ -1,10 +1,9 @@
 import _ from 'intl'
-import ActionRowButton from 'action-row-button'
+import SortedTable from 'sorted-table'
```

Comment:
These changes shouldn't be here and belong to the PR #2448. You should always [create a dedicated branch](https://gitlab.com/vates/company/wikis/development-process#create-a-development-branch) to develop a feature to avoid this problem.